### DISCUSSION TITLE
Renaming overrides for C# convention

### DISCUSSION THREAD

**Discussion by PerfectlyFineCode** [godotengine/godot-proposals#9201] [Scripting]
I saw that method overrides starts with an underscore, that goes against C# convention. Methods should be PascalCase and should not include characters such as `_`.

So my suggestion is to rename `_Ready` `_Process` and other APIs to fit the C# Convention, so `_Ready` would translate to `Ready`.

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/757.
